### DISCUSSION TITLE
6LBR

### DISCUSSION THREAD

**Discussion by MfaizR** [contiki-ng/contiki-ng#2982] [Q&A]
Hi, anyone working 6LBR with Contiki-NG?

**Comment by chansdad** [comment]
havent seen anyone work on it .  I am looking at using 6lbr as the base for a broder router

**Reply by MfaizR** [reply]
I can see that no more 6LBR development activity after 2018. I wish somebody could help me to compile the 6LBR with Contiki-NG. I have tried to compile but failed.

**Reply by chansdad** [reply]
This  app is dated . I never liked the interface , i am at a point now i want to pay someone to write a totally new border router interface for me.

**Comment by MfaizR** [comment]
manage to solved the 6LBR compilation.
